stage1/init/common/pod.go

```diff
@@ -1105,3 +1112,139 @@ func protectSystemFiles(opts []*unit.UnitOption, appName types.ACName) []*unit.U
 	}
 	return opts
 }
+
+// getSeccompFilter gets an appc seccomp set and an optional error mode,
+// returning those values in a format suitable for systemd consumption.
+func getSeccompFilter(opts []*unit.UnitOption, p *stage1commontypes.Pod, isolators types.Isolators) ([]*unit.UnitOption, error) {
+	seccompIsolators := 0
+	seccompErrno := ""
+	charMode := ""
+	flag := ""
+	var err error
+	var seccompSet []string
+	for _, i := range isolators {
+		if seccomp, ok := i.Value().(types.LinuxSeccompSet); ok {
+			seccompIsolators++
+			// By appc spec, only one seccomp isolator per app is allowed
+			if seccompIsolators > 1 {
+				return nil, fmt.Errorf("too many seccomp isolators specified (%d)", seccompIsolators)
```

Comment:
%d will always be 2 because we return from the function before incrementing to 3 or more.